src/sentry/static/sentry/app/components/button.tsx

```diff
@@ -166,6 +168,7 @@ class Button extends React.Component<ButtonProps, {}> {
           size={size}
           priority={priority}
           borderless={borderless}
+          className={labelClassName}
```

Comment:
@doralchan @robinrendle How do we feel about being able to customize CSS for this subcomponent of `Button`?

Reply:
this would be the size of the button without the labelClass 
![image](https://user-images.githubusercontent.com/29228205/82249461-152b2480-994a-11ea-8a80-83e741bf23dc.png)

but it would look different of similar buttons 
![image](https://user-images.githubusercontent.com/29228205/82249514-31c75c80-994a-11ea-90cc-728e0160cecf.png)

https://sentry.io/organizations/sentry/issues/1341896039/?project=11276&query=is%3Aunresolved

Reply:
@priscilawebdev I think this button size is fine, the `[+]` is used in an entirely different part of the app